packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -256,6 +265,19 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
     onSidePaneIdChange?.(sidePaneRenderer?.id);
   }, [sidePaneRenderer?.id, onSidePaneIdChange]);
 
+  // Track the last dismissed errors of any error kind to prevent errors from re-appearing on subsequent page navigation
+  // This works by tracking the most recent timestamp of any active error type.
+  // And then tracking when that error type was last dismissed.
+  const activeErrors = usePropsFor(ErrorBar).activeErrorMessages;
+  const [trackedErrors, setTrackedErrors] = useState<TrackedErrors>({} as TrackedErrors);
+  useEffect(() => {
+    setTrackedErrors((prev) => updateTrackedErrorsWithActiveErrors(prev, activeErrors));
```

Comment:
What do the tracked errors signify and are these internal to the call composite. Shouldn't we only have active and dismissed errors maintained globally?

Reply:
TrackedErrors is the a mapping of any errors that have been triggered to information about the timestamp when they were last active, and the timestamp when it was last dismissed. `trackedErrors` maps the error type to an object containing:
```
  mostRecentlyActive: Date;
  lastDismissedAt?: Date;
```

So, for example, when a new error comes in, that error will be added to set:
```
trackedErrors = {
  newError1: {
    lastActive: Date.now()
  }
}
```

Then if another error comes in:
```
trackedErrors = {
  newError1: {
    lastActive: <oldTimestampA>
  },
  newError2: {
    lastActive: Date.now()
  }
}
```

Then if error1 is dismissed but is still active (e.g. no headphones are still plugged in but the error was dismissed):
```
trackedErrors = {
  newError1: {
    lastActive: <oldTimestampA>,
    dismissedAt: Date.now()
  },
  newError2: {
    lastActive: <oldTimeStampB>
  }
}
```

If error1 is resolved and no longer exists in stateful, its removed from the set of errors being tracked:
```
trackedErrors = {
  newError2: {
    lastActive: <oldTimeStampB>
  }
}
```

If it reappears again, it gets added as normal, and will reappear to the user to be dismissed again:
```
trackedErrors = {
  newError1: {
    lastActive: Date.now()
  },
  newError2: {
    lastActive: <oldTimeStampB>
  }
}
```

Reply:
The tracking is local to the CallComposite, if someone plugged the same adapter into a new CallComposite the errors would reappear to be dimissed